<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventPage</class>
 <widget class="QWidget" name="EventPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Trigger</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="3" column="0">
         <widget class="QRadioButton" name="radio_autorun">
          <property name="text">
           <string>Autorun</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radio_event_touch">
          <property name="text">
           <string>Event Touch</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radio_player_touch">
          <property name="text">
           <string>Player Touch</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radio_action_button">
          <property name="text">
           <string>Action Button</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="radio_parallel_process">
          <property name="text">
           <string>Parallel Process</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Conditions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QCheckBox" name="check_switch2">
          <property name="text">
           <string>Switch</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="combo_switch2"/>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="check_switch1">
          <property name="text">
           <string>Switch</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="check_self_switch">
          <property name="text">
           <string>Self Switch</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="combo_variable"/>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spin_variable">
          <property name="minimum">
           <number>-9999999</number>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="check_variable">
          <property name="text">
           <string>Variable</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="combo_switch1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>or above</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>is ON</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>is ON</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>is</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="combo_self_switch">
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>D</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>is ON</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Autonomous Movement</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Move Route</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_5">
          <item>
           <property name="text">
            <string>Fixed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Random</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Approach</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Speed:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Freq:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox_6">
          <item>
           <property name="text">
            <string>1: Slowest</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2: Slower</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3: Slow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4: Fast</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5: Faster</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6: Fastest</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboBox_7">
          <item>
           <property name="text">
            <string>1: Lowest</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2: Lower</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3: Low</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4: High</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5: Higher</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6: Highest</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QCheckBox" name="check_move_animation">
          <property name="text">
           <string>Move Animation</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="check_through">
          <property name="text">
           <string>Through</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="check_stop_animation">
          <property name="text">
           <string>Stop Animation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="check_direction_fix">
          <property name="text">
           <string>Direction Fix</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="check_always_on_top">
          <property name="text">
           <string>Always on Top</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Graphics</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Event List</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="QListView" name="listView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
